### DISCUSSION TITLE
Created time & Updated time

### DISCUSSION THREAD

**Discussion by ghost** [Budibase/budibase#5859] [Help]
I was trying to update a column in Mysql table with updated timestamp when the record is updated and same for created row. 
In couch DB this is happening automatically, how to get this done for MYSQL database

**Comment by melohagan** [comment]
Hey @newtonlaw95 

This should be possible by creating a **created_at** and **updated_at** columns like so: https://stackoverflow.com/questions/50603953/how-to-add-created-at-and-updated-at-columns
